ProjectSample/projectsample/static/js/maincontroller.js

```diff
@@ -31,10 +31,16 @@ app.MainController = function(gettextCatalog, langUrlTemplate) {
   this.langUrlTemplate_ = langUrlTemplate;
 
   /**
-   * @private
+   * @type {string|undefined}
+   * @export
+   */
+  this.lang = undefined;
```

Comment:
We could use the empty string as the initial value, and just `string` for the type.